File: Aula 03 - Variaveis, Tipos de Dados e Operadores/cpp_logic/DesafioRoi.cpp
#include <iostream>
using namespace std;

int main() {
    double ganho, investimento;

    cout << "Digite o ganho: ";
    cin >> ganho;

    cout << "Digite o investimento: ";
    cin >> investimento;

    if (investimento == 0) {
        cout << "Erro: investimento nao pode ser zero." << endl;
    } else {
        double roi = ((ganho - investimento) / investimento) * 100;
        cout << "ROI: " << roi << "%" << endl;
    }

    return 0;
}

File: Aula 03 - Variaveis, Tipos de Dados e Operadores/python_logic/DesafioRoi.py
ganho = float(input("Digite o ganho: "))
investimento = float(input("Digite o investimento: "))

if investimento == 0:
    print("Erro: investimento não pode ser zero.")
else:
    roi = ((ganho - investimento) / investimento) * 100
    print(f"ROI: {roi:.2f}%")
